### PR TITLE
Cleaned up README

### PR DESCRIPTION
**Pull request by agentFinland**
- corrected minor grammar errors mainly where words were missing or in wrong order
- corrected some inconsistant styling
- changed some formatting of "What's great about Steward" section to improve readability
- clarified a few steps in "Getting Started" that had me confused during my first Steward install

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,73 +7,82 @@
...
+   - downloads and installs Selenium server with one command
+   - sets-up browser of your choice
+   - automatically takes a screenshot on failed assertions
+   - produces test results in JUnit format (easily processable e.g. by Jenkins and other tools)
+   - and more...
+- Tests are run in parallel, so the only bottleneck is the number of Selenium nodes you start simultaneously
+- Simple syntax sugar layer on top of default [WebDriver commands](https://github.com/facebook/php-webdriver/wiki/Example-command-reference) helps shorten your tests and improve readability
+- If you already use PHP, you don't have to learn a new language to write functional tests. Moreover, if you are familiar with unit tests and PHPUnit, you know it all
+- Allows you to plan test dependencies
+   - Ex. need to wait 2 minutes until some event gets through your message queue before testing the result? No problem! The order of tests is optimized to minimize the total execution time
```
Comment (OndraM) [inline README.md]: Eg? Or if this is shortcut for example, I'd prefer longer version without the shortcut ("For example ..."). Also the same case below.

File: README.md [hunk 2]
```diff
@@ -7,73 +7,82 @@
...
+- Status of tests can be clearly watched during test execution, so you will easily know how many tests have finished and what their results are
+- You can extend Steward easily by registering custom events to EventDispatcher.
+   - Ex. you can add custom configuration options or change parameters passed to child PHPUnit processes
+- Cloud services like [Sauce Labs](https://saucelabs.com/), [BrowserStack](https://www.browserstack.com/) or [TestingBot](https://testingbot.com/) are fully integrated giving you a chance to run tests with less setup and without your own infrastructure
+- Steward is field tested - we use it daily in [our company](https://www.lmc.eu/english) to maintain the quality of our products thanks to hundreds of test-cases
+   - the library itself is also extensively covered by unit tests
```
Comment (OndraM) [inline README.md]: I'd keep this as part of the sentence, ie. not as standalone point.

File: README.md [hunk 3]
```diff
@@ -7,73 +7,82 @@
...
-### 2. Download Selenium server and browser driver
-You can download and run Selenium standalone server and the browser locally right on your computer.
-Another possibility is to [start Selenium server + browser inside Docker container][wiki-docker].
-To run Selenium locally:
+### 2. Download Selenium Server and browser drivers
+You can download and run Selenium Standalone Server with the test browser opening locally right on your computer.
+Another possibility is to [start Selenium Server and test browser inside a Docker container][wiki-docker].
 
-#### Get Selenium standalone server
+####To run Selenium locally:
```
Comment (OndraM) [inline README.md]: This is not properly formatted for markdown. And I think this header is unnecessary - better would IMHO be to reword previous sentences to something like `This step only applies if you want to download and run Selenium Standalone Server with the test browser locally right on your computer. Another possibility is to [start Selenium Server and test browser inside a Docker container][wiki-docker].`.

What do you think, would it be understandable?

**Review by OndraM — CHANGES_REQUESTED** [review]
Hi @agentFinland , this is really a great contribution! This kind of feedback on readability of the readme from newcomers is extremely valuable, so thank you very much for it.

I have few minor points, could you please have a look?

Also - I would prefer keeping periods on end of the sentences (which begins with a capital letter) in bullet point lists. Could you change this, please?

File: README.md (hunk 3 above)
Comment (agentFinland) [reply README.md]: I agree with this. I'll make the change.

**Comment by agentFinland** [comment]
I agree with the suggested changes and have updated my pull request accordingly. Thanks again @OndraM for the suggestions.

**Pull request merged**